README.md

```diff
@@ -25,4 +25,14 @@ Run test:
 npx truffle test
 ```
 
-Use scripts as described in `scripts/README.md`.
\ No newline at end of file
+Use scripts as described in `scripts/README.md`.
+
+## Synthetix Spread Orders
+
+This service requires a service that runs a script every 5 minutes. We have configured this to run inside a Docker container that can be interacted with as follows:
+
+```sh
+docker build -t gnosispm/dex-liquidity-provision .
+ export PK=<YOUR_PRIVATE_KEY>
```

Comment:
When running locally PK needs to be passed into the run command via the `-e` argument right?